MainHomework/MainHomework/AllCharactersCollectionViewCell.swift

```diff
@@ -0,0 +1,133 @@
+    }
+    
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            characterImageView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            characterImageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            characterImageView.heightAnchor.constraint(equalTo: contentView.heightAnchor),
+            characterImageView.widthAnchor.constraint(equalTo: contentView.heightAnchor),
+            
+            nameLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 8),
+            nameLabel.leadingAnchor.constraint(equalTo: characterImageView.trailingAnchor, constant: 8),
+            nameLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -8),
+            
+            colorIndicatorView.topAnchor.constraint(equalTo: nameLabel.bottomAnchor, constant: 17),
+            colorIndicatorView.leadingAnchor.constraint(equalTo: characterImageView.trailingAnchor, constant: 8),
+            colorIndicatorView.heightAnchor.constraint(equalToConstant: 8),
+            colorIndicatorView.widthAnchor.constraint(equalTo: colorIndicatorView.heightAnchor),
+            
+            statusLabel.topAnchor.constraint(equalTo: nameLabel.bottomAnchor, constant: 10),
+            statusLabel.leadingAnchor.constraint(equalTo: colorIndicatorView.trailingAnchor, constant: 8),
+            statusLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -8),
+            statusLabel.centerYAnchor.constraint(equalTo: colorIndicatorView.centerYAnchor),
+            
+            typeLabel.topAnchor.constraint(equalTo: statusLabel.bottomAnchor, constant: 4),
+            typeLabel.leadingAnchor.constraint(equalTo: characterImageView.trailingAnchor, constant: 8),
+            typeLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -8),
+            
+            commonLocationLabel.topAnchor.constraint(equalTo: typeLabel.bottomAnchor, constant: 4),
+            commonLocationLabel.leadingAnchor.constraint(equalTo: characterImageView.trailingAnchor, constant: 8),
+            commonLocationLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -8),
+            
+            locationLabel.topAnchor.constraint(equalTo: commonLocationLabel.bottomAnchor, constant: 2),
+            locationLabel.leadingAnchor.constraint(equalTo: characterImageView.trailingAnchor, constant: 8),
+            locationLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -8)
+        ])
+    }
+    
+    func configure(result: Hero) {
+        setupCell()
+        setupConstraints()
+        
+        guard let imageURL = URL(string: result.image ?? " ") else { return }
+        characterImageView.load(url: imageURL)
+        
+        nameLabel.text = result.name
+        statusLabel.text = "\(result.status?.rawValue ?? "  ") - \(result.species?.rawValue ?? "  ")"
```

Comment:
```suggestion
        statusLabel.text = "\(result.status?.rawValue ?? "") - \(result.species?.rawValue ?? "")"
```
Лучше так, поправь везде